v1/ast/oracle/oracle.go

```diff
@@ -22,11 +21,21 @@ func (e Error) Error() string {
 // Oracle implements different queries over ASTs, e.g., find definition.
 type Oracle struct {
 	compiler *ast.Compiler
+	manager  *find.Manager
 }
 
 // New returns a new Oracle object.
 func New() *Oracle {
-	return &Oracle{}
+	manager := find.NewManager()
+
+	manager.Register(find.NewRefLocator())
```

Comment:
I don't think your refactoring actually changes this behaviour, but I find it strange that we lookup refs before vars. E.g. if a user queries the location of a var that happens to be inside a ref (except for the first ref term), then I think they'd expect to find the definition of that var, not the ref it happens to be inside.
Or am I misunderstanding this?